modules/installation/proc-install-rhdh-ocp-operator.adoc

```diff
@@ -19,7 +19,7 @@ For more information about creating a project in {ocp-short}, see link:https://d
 . From the *Developer Catalog* panel, click *Operator Backed*.
 . In the *Filter by keyword* box, enter _{product-short}_ and click the *{product}* card.
 . Click *Create*.
-. Add <<Configuring the {product-short} Custom Resource, custom configurations>> for the {product} instance.
+. Add <<Adding a custom application configuration file to {ocp-short} using the Operator, custom configurations>> for the {product} instance.
```

Comment:
Couldn't find this in the preview to check the link.

Reply:
This is in the Getting Started guide, I think. Just a little xref fix since we deleted the CR config content that was being referenced here.